kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -42,18 +42,13 @@ class APIDataset(AbstractDataset[None, requests.Response]):
         >>>
         >>>
         >>> dataset = APIDataset(
-        ...     url="https://quickstats.nass.usda.gov",
+        ...     url="https://api.spaceflightnewsapi.net/v4/articles",
```

Comment:
We can use something even more mission critical so it never goes down 😄 https://celestrak.org/NORAD/elements/gp.php?INTDES=2020-025&FORMAT=JSON-PRETTY

Reply:
_Nothing is permanent_